source/mainnet/net/mobile-wallet/update-baker-mw.rst

```diff
@@ -39,110 +51,593 @@ Once you have access baker settings for your wallet, you have four options:
     - Update baker keys
     - Stop baking
 
-.. image:: ../images/mobile-wallet/baker-hamburger-menu.png
-    :width: 25%
+.. _update-baker-stake-mw:
 
-Update baker stake
-==================
+Update baker stake and restaking preference
+===========================================
 
-If you choose **Update baker stake** you see your balance and the current Baker stake amount. Enter the amount you want to stake. This is the full amount to stake and is not added to the existing stake. You can also choose to adjust your restake setting between **Yes, restake** rewards or **No, don’t restake** rewards. Tap **Continue**.
+You can change the :ref:`staked amount <concepts-baker-stake>` on a baker account except during a :ref:`cool-down period<glossary-cool-down-period>`. If you increase the stake, the new stake takes effect at the next :ref:`pay day<glossary-pay-day>`. If the change is made in the last epoch before pay day, then the change will not occur until the following pay day. However, if you decrease the stake, there is a longer cool-down period of three weeks before the new stake is applied. During this period, you'll not be able to remove the baker account or further update the stake. After the cool-down period, the amount you’ve decreased the stake with is returned to your disposable balance at the next pay day.
 
-.. image:: ../images/mobile-wallet/update-baker-stake-amt-mw.png
-    :width: 25%
+When you change the stake it can influence your :ref:`chance of being selected to bake a block<glossary-winning-probability>` and receive baker rewards and of being included in the :ref:`finalization <glossary-finalization>` committee. If you decrease the stake, you decrease your chances of baking blocks and of being included in the finalization committee. Likewise, if you increase the stake, you increase your chances of baking a blocks and of being included in the finalization committee.
```

Comment:
```suggestion
When you change the stake it can influence your :ref:`chance of being selected to bake a block<glossary-winning-probability>` and receive baker rewards and of being included in the :ref:`finalization <glossary-finalization>` committee. If you decrease the stake, you decrease your chances of baking blocks and of being included in the finalization committee. Likewise, if you increase the stake, you increase your chances of baking a block and of being included in the finalization committee.
```